mempool/src/feerate.rs

```diff
@@ -0,0 +1,56 @@
+use common::primitives::amount::Amount;
+
+use crate::error::TxValidationError;
+
+lazy_static::lazy_static! {
+    pub(crate) static ref INCREMENTAL_RELAY_FEE_RATE: FeeRate = FeeRate::new(1000);
```

Comment:
Has the 1_000 come from somewhere?

Reply:
Bitcoin Core's DEFAULT_MIN_RELAY_TX_FEE (validation.h).